blockdata/datasaver.go

```diff
@@ -252,15 +252,13 @@ func JSONFormatBlockData(data *BlockData) (*bytes.Buffer, error) {
 	return &jsonAllIndented, err
 }
 
-// BlockTrigger wraps a simple function of builtin-typed hash and height that
-// reports no error.
+// BlockTrigger wraps a simple function of builtin-typed hash and height.
 type BlockTrigger struct {
-	Saver func(string, uint32)
+	Saver func(string, uint32) error
```

Comment:
Unfortunately, another breaking change, which bumps the major revision again, to v3.0.0